includes/Modules/Sign_In_With_Google.php

```diff
@@ -381,7 +381,7 @@ private function render_signinwithgoogle() {
 		google.accounts.id.renderButton( parent, <?php echo wp_json_encode( $btn_args ); ?> );
 	<?php endif; // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
 
-	<?php if ( $settings['oneTapEnabled'] ) : // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
+	<?php if ( $settings['oneTapEnabled'] && ! is_user_logged_in() ) : // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
```

Comment:
@tofumatt we also need to check that we are not on the login page because even if you are already logged in and go to the login page, you still can login using different account.

Reply:
Ah, true, thanks 👍🏻